CHANGELOG.md

```diff
@@ -1,11 +1,40 @@
 # Change Log
 
-TBD:
-  - Merged wgpu-rs and wgpu back into a single repository
+## v0.9 (2021-06-18)
+  - Merged wgpu-rs and wgpu back into a single repository.
+  - Updated naga to use `v0.5`.
+  - Added `Features::VERTEX_WRITABLE_STORAGE`.
+  - Added `Features::CLEAR_COMMANDS` which allows you to use `cmd_buf.clear_texture` and `cmd_buf.clear_buffer`.
+  - `Buffer::get_mapped_range` can now have a range of zero.
+  - Added shader label to log messages if validation fails.
+  - Fixed output spirv requiring the "kernal" capability.
+  - Fixed segfault due to improper drop order.
+  - Fixed incorrect dynamic stencil reference for Replace ops.
+  - Stopped unconditionally adding cubemap flags when the backend doesn't support cubemaps.
+  - Updated default storage buffer/image limit to `8` from `4`.
+  - Validation: Ensure that if resources are viewed from the vertex stage, they are read only unless `Features::VERTEX_WRITABLE_STORAGE` is true.
```

Comment:
can we group them, i.e.
```
  - Validation:
    -  A
    -  B
```

Same for "Fixes:" and "Infrastructure:" (or "Internal:")

Reply:
Sure, fixed!